For musl libc it is required to include <sys/reg.h> to
have __WORDSIZE defined to e.g. 32 for arm*-musl.

---
 src/install/util.h | 3 +++
 1 file changed, 3 insertions(+)

--- a/src/install/util.h
+++ b/src/install/util.h
@@ -33,6 +33,9 @@
 #include <signal.h>
 #include <sched.h>
 #include <limits.h>
+#if !defined(__GLIBC__)
+#include <sys/reg.h>
+#endif
 #include <sys/stat.h>
 #include <dirent.h>
 #include <sys/resource.h>
-- 
2.37.3

